lib/packages.mts

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
For the time being I had to create 2 files. `lib/packages.mts` and `lib/packages.ts`. They have the same content, but are needed because of ts-node, and local-bootstrap.